src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java

```diff
@@ -0,0 +1,62 @@
+package NewBalance.Pages;
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class NewBalanceLogInAndRegistrationPage extends BasePage {
+    public NewBalanceLogInAndRegistrationPage(WebDriver driver) {
+        super(driver);
+    }
```

Comment:
add empty lines before and after the constructor (9&10 and 12&13 lines)

Reply:
fixed